docs/mkdocs.yml

```diff
@@ -198,7 +198,6 @@ nav:
         - task get: 'cmd/planner/task/task-get.md'
         - task list: 'cmd/planner/task/task-list.md'
         - task set: 'cmd/planner/task/task-set.md'
-        - task details get: 'cmd/planner/task/task-details-get.md'
```

Comment:
Let's keep the entry but have it point to the `task get` page for easier discovery.